dev/com.ibm.ws.microprofile.rest.client_fat/fat/src/com/ibm/ws/microprofile/rest/client/fat/JsonbContextTest.java

```diff
@@ -33,7 +33,8 @@
 
     @ClassRule
     public static RepeatTests r = RepeatTests.withoutModification()
-        .andWith(FATSuite.MP_REST_CLIENT_WITH_CONFIG("1.3", SERVER_NAME));
+        .andWith(FATSuite.MP_REST_CLIENT_WITH_CONFIG("1.3", SERVER_NAME))
+        .andWith(FATSuite.MP_REST_CLIENT_WITH_CONFIG("1.4", SERVER_NAME));
```

Comment:
Just checking... is it right that this test does not use `FeatureReplacementAction.EE8_FEATURES()` like the others do?

Reply:
Yes, that is correct.  JSON-B is only in EE8, so we don't need to run it again with any other EE level (until EE9...).